crates/viewer/re_component_ui/src/resolution.rs

```diff
@@ -0,0 +1,13 @@
+use re_format::format_f32;
+use re_types::components::Resolution;
+use re_viewer_context::{MaybeMutRef, UiLayout, ViewerContext};
+
+pub fn edit_or_view_resolution(
+    _ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    value: &mut MaybeMutRef<'_, Resolution>,
+) -> egui::Response {
+    // Don't allow editing resolution for now.
```

Comment:
Why not? If it is just lack of time then we should link to a ticket with a TODO, for all of these cases

Reply:
todo to self: link the transform unsupported one in